service_info/templates/cms/content-types/faq.html

```diff
@@ -0,0 +1,78 @@
+{% extends "cms/base.html" %}
+{% load cms_tags sekizai_tags staticfiles %}
+
+{% block content %}
+    {% addtoblock "css" %}
+        <link rel="stylesheet" type="text/less" media="all" href="{% static 'less/content-types/faq.less' %}">
+    {% endaddtoblock %}
+    
+    <ul id="faq">
+        {% for qa in qas %}
+            <li class="qa">
+                <div class="q">
+                    {{ qa.q }}
+                </div>
+                <div class="a">
+                    {{ qa.a }}
+                </div>
+            </li>
+        {% endfor %}
+
+<!-- Some dummy FAQ items. -->
```

Comment:
I was able to change my page to use this template, but I couldn't figure out how to add my own Q's and A's. I assume that will come in a later ticket?